packages/web-app-files/tests/unit/components/SideBar/Shares/FileShares.spec.js

```diff
@@ -78,19 +77,6 @@ describe('FileShares', () => {
       expect(wrapper).toMatchSnapshot()
     })
 
-    it('can toggle the collaborators list by clicking the avatar wrapper button', async () => {
```

Comment:
We should actually re-write the test because toggling is still available, just with a different logic :)